list-sensors.cpp

```diff
@@ -465,8 +504,16 @@ int main(int argc, char* argv[])
     }
     catch (const sdbusplus::exception::SdBusError& ex)
     {
-        fprintf(stderr, "Error: %s\n", ex.what());
-        return EXIT_FAILURE;
+        if (strstr(ex.what(), "org.freedesktop.DBus.Error.FileNotFound"))
```

Comment:
`SdBusError::name()` returns the original exception class.
Likely, the code below would be a little bit better.
```cpp
if (0 == strcmp(ex.name(), "org.freedesktop.DBus.Error.FileNotFound"))
```